app/views/home.njk

```diff
@@ -67,6 +67,9 @@
         <div class="govuk-divider"></div>
         <h1 class="govuk-heading-m">Account options</h1>
         <ul class="govuk-list">
+          <li>
+            <a class class="govuk-link" href="/picker">Choose organisation</a>
```

Comment:
I think we should change this to `/landing-page/picker`.  By default anything not prefixed with `/auth` or `/business` will be redirected to `/landing-page/whatevs` anyway, but just to avoid a potential conflict in future if say we have a `/picker` service.